### ISSUE TITLE
Usage statistics

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/ecological-science#14]
- stats on data usage from ALA. For the ALA to get stats on what data are used in the BCCVL. Downloads and what analyses are being done would be ideal.

**Comment by Tasilee** [comment]
I agree. It would be interesting to know what species are being modelled in the BCCvL. This aligns with something that I had been considering for some time - now we have points, expert distributions, (soon) tracks but we don't capture 'metadata' in any standard form regarding use of MaxEnt in the Spatial Portal for modeling.  In short, wouldn't it be nice when searching for species that you could see how many models were done (and who did them).
